src/logic/notifications/store/notifications.ts

```diff
@@ -66,9 +71,20 @@ export const showNotification = (
     return key
   }
 }
-export const closeNotification = createAction<ClosePayload>(NOTIFICATION_ACTIONS.CLOSE)
+export const readNotification = createAction<KeyPayload>(NOTIFICATION_ACTIONS.READ)
+export const closeNotification = (payload: ClosePayload): ThunkAction<void, AppReduxState, undefined, AnyAction> => {
+  return (dispatch): void => {
+    const { read = true } = payload
+    if (read) {
+      dispatch(readNotification(payload))
+    }
```

Comment:
If its always set to true can we remove the condition?

Reply:
`true` is it's default value if no `read` property is passed in the payload.